internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -711,7 +711,7 @@ func (r *AgentServiceConfigReconciler) newAssistedServiceDeployment(log logrus.F
 	annotations := instance.ObjectMeta.GetAnnotations()
 	configmapName, ok := annotations[configmapAnnotation]
 	if ok {
-		log.Infof("ConfigMap %v being used to configure assisted-service deployment", configmapName)
+		log.Infof("ConfigMap %s being used to configure assisted-service deployment", configmapName)
```

Comment:
What do you think about printing the namespace of the configmap as well ?

Reply:
Sure, good idea 👍🏻